pkg/kubectl/describe.go

```diff
@@ -87,6 +87,7 @@ func describerMap(c *client.Client) map[unversioned.GroupKind]Describer {
 		api.Kind("Endpoints"):             &EndpointsDescriber{c},
 		api.Kind("ConfigMap"):             &ConfigMapDescriber{c},
 
+		extensions.Kind("ReplicaSet"):              &ReplicaSetDescriber{c},
```

Comment:
@smarterclayton @nikhiljindal DefaultObjectDescriber (below) was added by PR #4880. 

DefaultObjectDescriber only supports a small subset of resource types. I can't see it used anywhere other than the DefaultObjectDescriber unit test. I checked Openshift Origin also.

That PR was about enabling printing of local objects. I don't see how DefaultObjectDescriber is related to that. Can we rip it out?